packages/js/product-editor/src/components/variations-table/variation-actions-menu/variation-actions-menu.tsx

```diff
@@ -85,6 +85,7 @@ export function VariationActionsMenu( {
 				onClick() {
 					recordEvent( 'product_variations_menu_view', {
 						source: TRACKS_SOURCE,
+						variationId: variation.id,
```

Comment:
This is an invalid prop format for tracks, it should be `variation_id`, see the error message:
<img width="705" alt="Screenshot 2023-08-25 at 9 04 25 AM" src="https://github.com/woocommerce/woocommerce/assets/2240960/0f43daaf-3c6c-4a84-ab01-b01c0340893d">
You can also check this in the rejected track events, it will reject it with: `eventname or eventprops failed "^[a-z_][a-z0-9_]*$" pattern`

Reply:
I also am not sure how beneficial it is to pass the `variationId` into tracks, I guess maybe only if we want to see how many updates are being done on a single variation.